include/cuda/std/barrier

```diff
@@ -34,6 +34,11 @@ _LIBCUDACXX_BEGIN_NAMESPACE_CUDA
 #  error "CUDA synchronization primitives are only supported for sm_70 and up."
 #endif
 
+_LIBCUDACXX_INLINE_VISIBILITY
+constexpr bool __is_bulk_enabled() {
+    return true;
+}
```

Comment:
My question would be why it is a function instead of a variable template or trait?

I assume we do not want a user to specialize that
```suggestion
_LIBCUDACXX_INLINE_VAR constexpr bool __is_bulk_enabled = true;
```